src/modules.js

```diff
@@ -447,7 +447,10 @@ function exportRuntime() {
       }
     }
 
-    if (unexported.length) {
+    // When using MINIMAL_RUNTIME, MODULARIZE, and EXPORT_ALL,
+    // the symbols will be assigned to the `Module` object, but only **after**
+    // this code.
+    if ((!MINIMAL_RUNTIME || !EXPORT_ALL) && unexported.length) {
```

Comment:
I think we still want to support unexportedRuntimeSymbols, so we should find a way to restructure or reorder such that this keeps working.

Reply:
Do we need to in case? Because it's specifically disabled for `EXPORT_ALL`.

Reply:
Oh I see, perhaps the whole purpose of unexportedRuntimeSymbols is mute in `EXPORT_ALL` mode.

Perhaps we should change line 434 above to `if (ASSERTIONS && !EXPORT_ALL) {` above?   That might be good to land as a separate change with a separate test since it doesn't just effect minimal runtime.  As part of that change we could also then wrap the declaration of `unexportedRuntimeSymbols` itself if `if (!EXPORT_ALL)`

Reply:
(I can take care of that you like).

Reply:
Maybe the first warning is actually useful?

Reply:
Yes, I'm actually not sure why that that first warning was inside this block at all.  Fixed that in #18567

Reply:
Okay great, I will rebase this PR. Do you want to make the change for export all or should I do it?